editor/block-toolbar/index.js

```diff
@@ -19,19 +20,51 @@ import './style.scss';
 import BlockSwitcher from '../block-switcher';
 import BlockMover from '../block-mover';
 import BlockRightMenu from '../block-settings-menu';
+import { isMac } from '../utils/dom';
```

Comment:
While it uses DOM properties to determine, `isMac` doesn't seem like a DOM-specific utility function.